packages/awsfargate/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: 0.2.2
+  changes:
+    - description: Set dimension fields.
```

This sets dimension fields, but also `agent.id`. Should we mention it here?

```suggestion
    - description: Set `dimension`, and `agent.id` fields.
```

---

`agent.id` was already present, but this time, it is just explicitly set on the `ecs.yml`. I can add that. What do you think is best @dmathieu?

---

Mentioning that it's now explicitly set sounds good to me.

---

Thanks @dmathieu , I updated the changelog to include that